src/main/resources/application.properties

```
@@ -1 +1,6 @@
+spring.datasource.url=jdbc:mysql://localhost:3306/rick_and_morty?serverTimeZone=UTC
+spring.datasource.username=root
+spring.datasource.password=1928374605BVss!@#$
```

it's better not to pass sensitive data and credentials to the open sources
```suggestion
spring.datasource.username=
spring.datasource.password=
```